internal/api/keppel/accounts_test.go

```diff
@@ -1685,6 +1715,15 @@ func TestDeleteAccount(t *testing.T) {
 
 	//as indicated by the response, we need to delete the specified manifest to
 	//proceed with the account deletion
+	assert.HTTPRequest{
+		Method: "DELETE",
+		Path: fmt.Sprintf(
+			"/keppel/v1/accounts/test1/repositories/%s/_manifests/%s",
+			repos[0].Name, "sha256:13493afd2a64d1a14fb5ca0d2fe312117239b935ea40dc35ca59c55931a86ab7",
+		),
+		Header:       map[string]string{"X-Test-Perms": "view:tenant1,delete:tenant1"},
+		ExpectStatus: http.StatusNoContent,
+	}.Check(t, r)
 	assert.HTTPRequest{
```

Comment:
I would like to see another test on `DELETE /keppel/v1/accounts/test1` to see that it now asks for the non-list manifest to be deleted.